package.json

```diff
@@ -17,9 +17,11 @@
     "lint:js": "eslint .",
     "start": "ember serve",
     "test": "ember test",
-    "test:all": "ember try:each"
+    "test:all": "ember try:each",
+    "prepublish": "ember ts:precompile"
```

Comment:
The default `ember-cli-typescript` blueprint adds the follow tasks to deal with publishing: 

```
    "prepublishOnly": "ember ts:precompile",
    "postpublish": "ember ts:clean"
```